src/pydicom/config.py

```diff
@@ -30,6 +30,50 @@ def __call__(
 _use_future = False
 _use_future_env = os.getenv("PYDICOM_FUTURE")
 
+
+# Logging system and debug function to change logging level
+logger = logging.getLogger("pydicom")
+logger.addHandler(logging.NullHandler())
+
+debugging: bool
+
+
+def debug(debug_on: bool = True, default_handler: bool = True) -> None:
+    """Turn on/off debugging of DICOM file reading and writing.
+
+    When debugging is on, file location and details about the elements read at
+    that location are logged to the 'pydicom' logger using Python's
+    :mod:`logging`
+    module.
+
+    Parameters
+    ----------
+    debug_on : bool, optional
+        If ``True`` (default) then turn on debugging, ``False`` to turn off.
+    default_handler : bool, optional
+        If ``True`` (default) then use :class:`logging.StreamHandler` as the
+        handler for log messages.
+    """
+    global logger, debugging
+
+    if default_handler:
+        handler = logging.StreamHandler()
+        formatter = logging.Formatter("%(message)s")
+        handler.setFormatter(formatter)
+        logger.addHandler(handler)
+
+    if debug_on:
+        logger.setLevel(logging.DEBUG)
+        debugging = True
+    else:
+        logger.setLevel(logging.WARNING)
+        debugging = False
+
+
+# force level=WARNING, in case logging default is set differently (issue 103)
```

Comment:
I wanted to ask how you dug up a ten-year-old issue before I noticed that the code just had been moved :)